File: abc/abc337/b.py
# https://atcoder.jp/contests/abc337/submissions/49591638

s = list(input())
t = []
for c in s:
  t.append(c)

t.sort()
for i in range(len(s)):
  if s[i] != t[i]:
    print('No')
    exit()

print('Yes')

File: abc/abc337/b.cs
// https://atcoder.jp/contests/abc337/submissions/49642735

using static System.Console;
using System.Text.RegularExpressions;

public class A
{
  static void Main()
  {
    string s = ReadLine();
    Write(Regex.IsMatch(s, "^A*B*C*$") ? "Yes" : "No");
  }
}

